src/main/java/com/blackducksoftware/integration/hub/api/view/MetaHandler.java

```diff
@@ -40,7 +40,7 @@ public MetaHandler(final IntLogger logger) {
     }
 
     public boolean hasLink(final HubView view, final String linkKey) throws HubIntegrationException {
-        final ResourceMetadata meta = view.meta;
+        final ResourceMetadata meta = view._meta;
```

Comment:
To remain consistent with the Hub you have removed the serialized name annotation, is that a correct assumption here?